core/src/main/java/org/jruby/runtime/ClassIndex.java

```diff
@@ -71,5 +71,5 @@
     CONTINUATION,
     BASICOBJECT,
     // insert new values here
-    MAX_CLASSES;
+    MAX_CLASSES, Layouts;
```

Comment:
No idea about this or the change to RubyObject. Must be some refactoring mess up. I reverted it.